### PR TITLE
Add .NET5 Squeezenet Sample

### PR DESCRIPTION
**Pull request by smk2007**
This sample demonstrates how to use .NET5 applications with the Inbox WinML and Redist WinML.

### CODE REVIEW

File: Samples/SqueezeNetObjectDetection/NET5/Labels.json [hunk 1]
```diff
@@ -0,0 +1,1002 @@
+{
+  "0": "tench, Tinca tinca",
```
Comment (martinb35) [inline Samples/SqueezeNetObjectDetection/NET5/Labels.json:2]: same as https://github.com/microsoft/Windows-Machine-Learning/blob/master/Samples/SqueezeNetObjectDetection/Desktop/cpp/Labels.txt?

can we share?

File: Samples/SqueezeNetObjectDetection/NET5/Properties/launchSettings.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "profiles": {
+    "SqueezeNetObjectDetectionNET5": {
+      "commandName": "Project",
+      "commandLineArgs": "SqueezeNet.onnx kitten_224.png cpu",
```
Comment (martinb35) [inline Samples/SqueezeNetObjectDetection/NET5/Properties/launchSettings.json:5]: do we do this in other samples? I think I like it, so it shows something useful right away. maybe we should add it elsewhere.

**Review by martinb35 — APPROVED** [review]

File: Samples/SqueezeNetObjectDetection/NET5/Properties/launchSettings.json (hunk 2 above)
Comment (smk2007) [reply Samples/SqueezeNetObjectDetection/NET5/Properties/launchSettings.json:5]: yeah, until now this file was part of the gitignore. We should add the relevant copy commands and launchSettings to each sample

File: Samples/SqueezeNetObjectDetection/NET5/Labels.json (hunk 1 above)
Comment (smk2007) [reply Samples/SqueezeNetObjectDetection/NET5/Labels.json:2]: We can, currently it is copied into each project. But squeezenet should be refactored to have all common assets pulled out.

**Pull request merged**